models/spleen_ct_segmentation/configs/multi_gpu_evaluate.json

```diff
@@ -15,14 +15,18 @@
     },
     "validate#dataloader#sampler": "@validate#sampler",
     "validate#handlers#1#_disabled_": "$dist.get_rank() > 0",
-    "evaluating": [
+    "initialize": [
         "$import torch.distributed as dist",
         "$dist.init_process_group(backend='nccl')",
```

Comment:
Please refer to my example in the ticket:
```
"$dist.is_initialized() or dist.init_process_group(backend='nccl')"
```
Same comment for the `multi_gpu_train`.

Thanks.